src/app/clients/ActionBar/Actions.tsx

```diff
@@ -0,0 +1,261 @@
+"use client";
+
+import React, { useState } from "react";
+import Menu from "@mui/material/Menu/Menu";
+import MenuList from "@mui/material/MenuList/MenuList";
+import MenuItem from "@mui/material/MenuItem/MenuItem";
+import dayjs, { Dayjs } from "dayjs";
+import { ClientsTableRow } from "@/app/clients/getClientsTableData";
+import ActionsModal, {
+    DayOverviewInput,
+    DriverOverviewInput,
+} from "@/app/clients/ActionBar/ActionsModal";
+import {
+    DayOverviewModalButton,
+    DriverOverviewModalButton,
+    ShippingLabelsModalButton,
+    ShoppingListModalButton,
+} from "@/app/clients/ActionBar/ActionsModalButton";
+import { SelectChangeEvent } from "@mui/material";
+
+const isNotMoreThanOne = (value: number): boolean => {
+    return value < 1;
+};
+
+const doesNotEqualsOne = (value: number): boolean => {
+    return value !== 1;
+};
+
+const doesNotEqualsZero = (value: number): boolean => {
+    return value !== 0;
+};
+
+type PdfType =
+    | "Download Shipping Labels"
+    | "Download Shopping List"
+    | "Download Driver Overview"
+    | "Download Day Overview";
+
+type AvailableActionsType = {
+    [pdfKey in PdfType]: {
+        showSelectedParcels: boolean;
+        errorCondition: (value: number) => boolean;
+        errorMessage: string;
+    };
+};
+
+const availableActions: AvailableActionsType = {
+    "Download Shipping Labels": {
+        showSelectedParcels: true,
+        errorCondition: isNotMoreThanOne,
+        errorMessage: "Please select at least 1 row for download.",
+    },
+    "Download Shopping List": {
+        showSelectedParcels: true,
+        errorCondition: doesNotEqualsOne,
+        errorMessage: "Please select only 1 row for download.",
+    },
+    "Download Driver Overview": {
+        showSelectedParcels: true,
+        errorCondition: isNotMoreThanOne,
+        errorMessage: "Please select at least 1 row for download.",
+    },
+    "Download Day Overview": {
+        showSelectedParcels: false,
+        errorCondition: doesNotEqualsZero,
+        errorMessage:
+            "The day overview will show the parcels for a particular date and location. It will show not the currently selected parcel. Please unselect the parcels.",
```

Comment:
Do we need this? Can we not just ignore selected parcels?

Reply:
I just thought that the user might expect the selected parcels to be shown rather than the day and location. I will leave the code in here as is but do agree that this could be discussed further in the future! <3